src/SerialLoops/Dialogs/BgmVolumePropertiesDialog.cs

```diff
@@ -0,0 +1,131 @@
+﻿using Eto.Forms;
+using HaruhiChokuretsuLib.Util;
+using NAudio.Wave;
+using NAudio.Wave.SampleProviders;
+using SerialLoops.Controls;
+using SerialLoops.Lib.Items;
+using SerialLoops.Lib.Util.WaveformRenderer;
+using SerialLoops.Utility;
+using SkiaSharp;
+using System.IO;
+
+namespace SerialLoops.Dialogs
+{
+    public class BgmVolumePropertiesDialog : Dialog
+    {
+        private ILogger _log;
+        private WaveStream _wav;
+        private StackLayout _waveformLayout;
+        private long _waveLength;
+
+        public bool SaveChanges { get; set; } = false;
+        public BgmVolumePreviewItem VolumePreview { get; set; }
+        public SKGuiImage Waveform { get; set; }
+        public SoundPlayerPanel VolumePreviewPlayer { get; set; }
+
+        public BgmVolumePropertiesDialog(WaveStream wav, string title, ILogger log)
+        {
+            _log = log;
+            Title = title;
```

Comment:
```suggestion
            Title = $"{title} - Adjust Volume";
```
Would just be nice to do this for consistency with other modals :)

Reply:
Yes sounds good!